prow/spyglass/lenses/metadata/metadata.ts

```diff
@@ -13,4 +13,10 @@ function toggleExpansion(dataId: string, textId: string, iconId: string): void {
   spyglass.contentUpdated();
 }
 
+function getLocalStartTime(): void {
+  const elem = document.getElementById("start_time")!;
+  elem.innerText = (new Date(elem.innerText)).toString();
+}
+
 (window as any).toggleExpansion = toggleExpansion;
+(window as any).onload = getLocalStartTime;
```

Comment:
This doesn't need the `as any` cast, because `onload` is a property that `window` is supposed to have.